src/index.ts

```diff
@@ -932,3 +1005,7 @@ export interface DatastoreOptions extends GoogleAuthOptions {
   apiEndpoint?: string;
   sslCreds?: ChannelCredentials;
 }
+
+export interface KeyToLegacyUrlsafeCallback {
+  (err?: Error | null, urlSafekey?: string): void;
```

Comment:
I'm not sure if there's an objective answer to this, but to me, `urlSafe` feels more correct than `urlsafe`. This and the previous line show two different interpretations of when to camelcase:

```
KeyToLegacyUrlsafeCallback
urlSafekey
```

I would expect:

```diff
- KeyToLegacyUrlsafeCallback
+ KeyToLegacyUrlSafeCallback

- urlSafekey
+ urlSafeKey
```

Reply:
Done!